src/server.ts

```diff
@@ -1,14 +1,24 @@
+/*
 const express = require("express");
 const app = express();
 const mongoose = require("mongoose");
 const cookieParser = require("cookie-parser");
 require("dotenv").config();
+*/
+
+import express from "express";
+import mongoose from "mongoose";
+import cookieParser from "cookie-parser";
+require("dotenv").config();
```

Comment:
Don't use require syntax with import. Import dotenv library and then call .config() on it